classloader/pom.xml

```diff
@@ -21,6 +21,14 @@
             <artifactId>asm</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.shrinkwrap</groupId>
+            <artifactId>shrinkwrap-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.shrinkwrap</groupId>
+            <artifactId>shrinkwrap-impl-base</artifactId>
+        </dependency>
```

Comment:
Aren't they both `<scope>test</scope>`? I'm pretty sure we wouldn't want all consumers to also transitively get ShrinkWrap.

Reply:
Right :) Thanks!